tests/Aspire.Workload.Tests/WorkloadTestsBase.cs

```diff
@@ -61,7 +61,7 @@ protected static async Task<ResourceRow[]> CheckDashboardHasResourcesAsync(IPage
             await Task.Delay(500);
 
             // _testOutput.WriteLine($"Checking for rows again");
-            ILocator rowsLocator = dashboardPage.Locator("//fluent-data-grid-row[@class='resource-row']");
+            ILocator rowsLocator = dashboardPage.Locator("//fluent-data-grid-row[@class='hover resource-row']");
```

Comment:
Hmm, I don't like that this requires the exact right classes be specified. Unrelated to this change though.

Reply:
Yeah it seems a little brittle. Maybe just switching on the row type instead might be better (to skip header rows)? Even that seems a little brittle. But I didn't even realize this existed until you pointed it out, so I wasn't trying to delve into it too far yet, just trying to unblock.

Reply:
I was hit by it last week. I vented here: https://github.com/dotnet/aspire/issues/4834

I'll add class name brittleness to the list